common/serialization/subtitles/subtitles.cpp

```diff
@@ -278,7 +282,10 @@ void parse_subtitle(const goos::Object& data,
         scene.m_sorting_group_idx = db.m_subtitle_groups->find_group_index(scene.m_sorting_group);
 
         for_each_in_list(entries, [&](const goos::Object& entry) {
-          if (entry.is_pair()) {
+          if (entry.is_empty_list()) {
+            lg::error("Empty list found in scene {}; skip it", scene.name());
+            return;
+          } else if (entry.is_pair()) {
```

Comment:
And I think you can get rid of this thing entirely because it shouldn't be possible to get an empty list in `for_each_in_list`.

Reply:
Well actually it is possible but it really shouldn't be happening so it should just fall back to the exception below.